docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1954,7 +1954,9 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/core/capabilityStatementCacheTimeout`|integer|The number of minutes that a tenant's CapabilityStatement is cached for the metadata endpoint. |
 |`fhirServer/core/extendedCodeableConceptValidation`|boolean|A boolean flag which indicates whether extended validation is performed by the server during object construction for code, Coding, CodeableConcept, Quantity, Uri, and String elements which have required bindings to value sets.|
 |`fhirServer/core/disabledOperations`|string|A comma-separated list of operations which are not allowed to run on the IBM FHIR Server, for example, `validate,import`. Note, do not include the dollar sign `$`|
-|`fhirServer/core/defaultPageSize`|integer|Sets the pageSize to use in search and history when no _count parameter is specified in the request. If a user-specified value exceeds the max page size (1000), then a warning is logged and max page size will be used. If not provided, the default page size (10) is used.|
+|`fhirServer/core/defaultPageSize`|integer|Sets the page size for search and history request results when no _count parameter is specified.|
+|`fhirServer/core/maxPageSize`|integer|Sets the max page size for search and history request results. If a user-specified _count parameter value exceeds the max page size, then a warning is logged and max page size will be used.|
+|`fhirServer/core/maxPageIncludeCount`|integer|Sets the max number of 'include' resources allowed per page for search and history request results. If a page of results from a search or history request will exceed the max number of 'include' resources, then an error will be returned in the request results.|
```

Comment:
```suggestion
|`fhirServer/core/maxPageIncludeCount`|integer|Sets the maximum number of 'include' resources allowed per page for search and history request results. If the number of 'include' resources returned for a page of results from a search or history request will exceed the number specified in `maxPageIncludeCount`, then an error will be returned in the request results.|
```

Reply:
Also, there's a paragraph in the Conformance guide in the 'Search parameters' section that talks about 'include' resources. It references the default max page size of 1000, so should be updated.

Reply:
Updated.